runtime/doc/plugin-guide.txt

```diff
@@ -0,0 +1,57 @@
+*plugin-guide.txt*                  Nvim
+
+                            NVIM REFERENCE MANUAL
+
+                          Guide to writing Nvim plugins
+
+
+                                       Type |gO| to see the table of contents.
+
+==============================================================================
+Introduction                                                      *plugin-guide*
+
+This guide will go through the basics of farming GitHub stars.
+
+Lua plugins are basically the same as a vim plugins, except the file extension
+is .lua instead of .vim and the file contains Lua code instead of vimscript.
```

Comment:
Even though that sounds facetious, I think this is actually a very useful point to make up front: Lua plugins are first and foremost Neovim plugins, so (some of) the same rules apply.

If you add to that the difference that Lua plugins use `lua` and `require('foo').bar` instead of Vimscript's `autoload` and `call foo#bar`, you are basically full correct; everything else is either

1. due to _language_ (or API) differences, or
2. generic advice for _any_ Neovim plugin.

Reply:
Good point.

Reply:
That's actually a great point. We should mention that at the intro, but perhaps in a simplified manner. Thanks.